pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -2595,3 +2603,15 @@ type Failover struct {
 	// +optional
 	RecoverByUID types.UID `json:"recoverByUID,omitempty"`
 }
+
+type ScalePolicy struct {
+	// ScaleInParallelism configures max scale in replicas for TiKV stores.
+	// +kubebuilder:default=1
+	// +optional
+	ScaleInParallelism *int32 `json:"scaleInParallelism,omitempty"`
```

Comment:
how about adding a validation?

Reply:
Sounds reasonable.

Reply:
@DanielZhangQD Should we treat `0` as a illegal argument too ?